### DISCUSSION TITLE
Using Outlines with vLLM/ FastChat

### DISCUSSION THREAD

**Discussion by rounak610** [outlines-dev/outlines#297] [Q&A] [answered]
How can we use outlines with vLLM or FastChat?

**Comment by brandonwillard** [comment]
We already have an open issue for vLLM, https://github.com/outlines-dev/outlines/issues/163, so feel free to follow and/or add to that.

Aside from that, exactly what kind of integrations with FastChat are you proposing?  It looks like they already have some [HF integrations](https://github.com/lm-sys/FastChat/blob/main/fastchat/serve/huggingface_api.py), so using those models with `outlines` could be a straightforward user-level integration.  If you can construct a minimal example demonstrating what you would like to be able to do, we can work from there.

**Comment by rlouf** [comment, marked as answer]
It is now possible to use with vLLM. Documentation coming.
